modules/standard/IO.chpl

```diff
@@ -2865,8 +2859,11 @@ record ioChar {
   }
 }
 
+deprecated "ioChar type is deprecated - please use :proc:`fileReader.readCodepoint` and :proc:`fileWriter.writeCodepoint` instead"
```

Comment:
Unless this is an exception for some reason, I think it's good to leave documentation intact on deprecated things until they are finally removed.